HW01 Part1.md

```diff
@@ -0,0 +1,80 @@
+
+In general, Theory III and Theory IV are less discussed and researched than Theory I and Theory II in the case of mass shooting, because the latter two are more common.
+
+#### Reference
+
+[1] Barry C L, McGinty E E, Vernick J S, et al. After Newtown—public opinion on gun policy and mental illness[J]. New England journal of medicine, 2013, 368(12): 1077-1081.
+
+[2] Metzl J M, MacLeish K T. Mental illness, mass shootings, and the politics of American firearms[J]. American journal of public health, 2015, 105(2): 240-249.
+
+[3]  ["Inside the mind of a mass murderer"](http://wane.com/2015/12/02/inside-the-mind-of-a-mass-murderer/). 
+
+[4] ["Sons of divorce, school shooters"](http://www.aei.org/publication/sons-of-divorce-school-shooters/). 
+
+[5] ["Orlando shooting: Obama condemns LGBT discrimination"](https://www.bbc.com/news/world-us-canada-36552461)
+
+[6] Larkin R W. The Columbine legacy: Rampage shootings as political acts[J]. American Behavioral Scientist, 2009, 52(9): 1309-1326.
+
+## Waiting until the last minute
+
+#### a.
+
+To explain why people tend to do things at the last minute, I refer to subjective and objective reasons. 
+
+Subjective reasons for procrastination include lack of self-confidence, low self-worth, and impatience. Many people don't get down to things until the last minute because they are not confident that they will do these things well. Some students may be afraid of writing a paper because they are afraid of screwing up and getting a bad grade. A related factor is low self-worth. Some people think too much before they actually do things, which is due to their pursuit for perfection, a byproduct of low self-worth. They value perfection too much because it's the only thing that can bring them satisfaction with themselves. As a result, they think, plan, and worry to get things done perfectly until the last minute, when they have no time to carry our their perfect plan. The third kind of person put off things because they are impatient. They can't stand the long period between starting to do something and finishing it. They want the fulfillment of finishing tasks shortly after they begin doing things so they choose to start at the last minute.
+
+Objective reasons are rooted in the characters of the tasks. A typical task that people tend to put off is long-term, uncertain, and difficult. Writting a paper and scientific research belong to this category. People are not willing to do this kind of task because they are more likely to fail and get no reward from doing it, and even if they succeed, the reward is too far away in the future. A more direct reason for procrastination is that people do not like the task and are forced to do it. In this case people get no reward after they finish it.
+
+#### b.
+
+Whether a person will postpone doing a task is determined by the following factors:
+
+1. This person's level of self-confidence: generally, how confident is this person that he/she will do things successfully.
```

Comment:
This model is already pretty complicated. Think about how to simplify it while still explaining the real-world behavior. It's a tricky balancing act